source/api_reference/search_payments_reference/index.html.md.erb

```diff
@@ -38,8 +38,8 @@ If your request has no query parameters, this endpoint returns all payments.
 | `email` | string | Returns payments with matching `email` values. You can use full or partial email addresses.<br><br>The `email` value is the paying user's email address. It was entered either by the service when the payment was created or by the paying user when they completed their payment.<br><br>You can [read more about prefilling payment fields](/optional_features/prefill_user_details/#prefill-payment-fields). |
 | `state` | string | Returns payments in a matching `state`. <br><br>`state` reflects where a payment is in the payment status lifecycle. You can [read more about the payment status lifecycle and the meaning of different `state` values](https://docs.payments.service.gov.uk/api_reference/#payment-status-lifecycle).<br><br>Available `state` values are:<br><br><ul> <li>`created`</li><li>`started`</li><li>`submitted`</li><li>`capturable`</li><li>`success`</li><li>`failed`</li><li>`cancelled`</li><li>`error`</li> |
 | `card_brand` | string | Returns payments paid with a particular card brand.<br><br>Available `card_brand` values are: <br><br><ul> <li>`american-express`</li><li>`diners-club`</li><li>`discover`</li><li>`jcb`</li><li>`maestro`</li><li>`master-card`</li><li>`unionpay`</li><li>`visa`</li>|
-| `from_date` | date (ISO 8601) | Returns payments created on or after the `from_date`.<br><br>Date and time must be coordinated Universal Time (UTC) and ISO 8601 format to second-level accuracy - `YYYY-MM-DDThh:mm:ssZ`. |
-| `to_date` | date (ISO 8601) | Returns payments created before the `to_date`.<br><br>Date and time must be coordinated Universal Time (UTC) and ISO 8601 format to second-level accuracy - `YYYY-MM-DDThh:mm:ssZ`. |
+| `from_date` | date (ISO 8601) | Returns payments created on or after the `from_date`.<br><br>Date and time must be coordinated Universal Time (UTC) and ISO 8601 format to second-level accuracy - `YYYY-MM-DDThh:mm:ss.sssZ`. |
+| `to_date` | date (ISO 8601) | Returns payments created before the `to_date`.<br><br>Date and time must be coordinated Universal Time (UTC) and ISO 8601 format to second-level accuracy - `YYYY-MM-DDThh:mm:ss.sssZ`. |
```

Comment:
This is an input value, not an output value, so we do allow millisecond-less input (for example, `2022-12-06T17:00:00Z` should work).

Reply:
Ah, good spot! This is what I get for CMD+F'ing in a hurry. Should be fixed now.